app-backend/batch/src/main/scala/stacExport/Utils.scala

```diff
@@ -235,7 +237,7 @@ object Utils {
       )
     }
     val sceneItemAbsolutePath =
-      s"$layerCollectionAbsolutePath/${sceneCollection.id}/${scene.id}.json"
+      s"$layerCollectionAbsolutePath/scenes/${scene.id}.json"
```

Comment:
A request from @jmorrison1847 

```suggestion
      s"$layerCollectionAbsolutePath/images/${scene.id}.json"
```

Reply:
I think this has to changes in at least two places if I remember right, so gonna do this locally and push

Reply:
It's a little confusing because we talk about a _scene_ collection, not an images collection -- should I change both?

Reply:
I think it makes sense to keep the verbiage in sync so switching to `images` collection would be good.